doc/ems-application-guide/src/ems-actuators/thermal-envelope.tex

```diff
@@ -19,6 +19,8 @@ \subsection{Window Shading Control}\label{window-shading-control}
   2.0:~~~~~~ Glazing is switched to a darker state (switchable glazings only).
 \item
   3.0: ~~~~~ Exterior shade is on.
+\item
+  4.0: ~~~~~ Exterior screen is on.
```

Comment:
You had mentioned there were additional values that could be added to the documentation.  Did you intend on adding more here?  I'm not saying you necessarily should but just wrapping it all up.

Reply:
@Myoldmopar Although @mjwitte suggested to add more values, the key choices for this field are limited, and I have not found a good way to allow more choices. You may merge this one first and don't close the issue, so that I may add more choices later as a separate PR.